### PR TITLE
Fix:update .bit name in nft-collection and bridge token

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------

## 1 
1-1、 support searching ends with `.bit`
1-2、the .bit domain tag on the account page should be a `link` to .bit account page

image

## 2
2-1、 showing magickbase.bit on hovering .bit icon is redundant, now showing text `.bit Name`

image

## 3
3-1、 token/holder list, magickbase.bit is not displayed ( `not include in this PR`)
https://v1.gwscan.com/token/133?tab=holders&id=133&page=13
3-2、 nft-collection, in the inventory, owner is not displayed by .bit name https://v1.gwscan.com/nft-collection/0x73c0e2a4b765ed81f63759d1fcf2c4dcd2925f4a?tab=inventory
3-3、nft-item, owner is not displayed with .bit name
https://v1.gwscan.com/nft-item/0x73c0e2a4b765ed81f63759d1fcf2c4dcd2925f4a/2375

image

Ref: https://github.com/Magickbase/godwoken_explorer/issues/1170

Check List
------

### Test
e2e Test
### Task
none

### CODE REVIEW

**Comment by FrederLu** [comment]
1、Now click to jump, but the text color is wrong.
image

2、The address whose .bit alias has been set needs to display the alias.
https://godwoken-explorer-oklw14yhl-magickbase.vercel.app/token/84440?tab=holders
image

**Comment by JeffreyMa597** [comment]
> 1、Now click to jump, but the text color is wrong. image

updated.

**Comment by FrederLu** [comment]
image
You can see the corresponding data display below.

1、The corresponding data is not displayed.
https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=holders
image
2、No alias information displayed yet.
https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory
image
https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18
image

**Comment by JeffreyMa597** [comment]
> 1、The corresponding data is not displayed. https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=holders 

fixed.

2、No alias information displayed yet. https://godwoken-explorer-lf9snmtks-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory 

waiting for the update from our endpoint.

**Comment by FrederLu** [comment]
https://godwoken-explorer-2c6ln4cjd-magickbase.vercel.app/token/84440?tab=holders
image
Just to make sure, there are still some changes to be made here.

**Comment by JeffreyMa597** [comment]
> https://godwoken-explorer-2c6ln4cjd-magickbase.vercel.app/token/84440?tab=holders image Just to make sure, there are still some changes to be made here.

@FrederLu  @Naupio I wondered whether I can submit other changes first with this PR except the changes in the `holders` tab. We still waiting for the update from our endpoint to change the `http` request to `Graphql` request. Thank you.

**Comment by FrederLu** [comment]
@JeffreyMa597  The following questions need to be confirmed:
https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18
1、On the above page, changing the page size will distort the icons.
image

2、After `Owner` is changed to an alias, the alignment with `TokenId` needs to be fine-tuned.
https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory
image

**Comment by JeffreyMa597** [comment]
> @JeffreyMa597 The following questions need to be confirmed: https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18 1、On the above page, changing the page size will distort the icons. 

Fixed.
image
 


> 2、After `Owner` is changed to an alias, the alignment with `TokenId` needs to be fine-tuned. https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/nft-collection/0x0772997bfde614faa246b3cc98358f459d7911f7?tab=inventory 

Fixed. 
image


> 3、There are addresses with aliases under different tabs on the same page, but the display is abnormal. (A)https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/multi-token-item/0xfa4d45c9b89d8b8025ca7a5c8aab15ba03f32db9/4?tab=holders 

Fixed.
image

 
> (B)https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/multi-token-item/0xfa4d45c9b89d8b8025ca7a5c8aab15ba03f32db9/4?tab=inventory 

Fixed.
image

 
> 4、There is no data in holders. https://godwoken-explorer-9mgu7gvzq-magickbase.vercel.app/multi-token-collection/0xfa4d45c9b89d8b8025ca7a5c8aab15ba03f32db9?tab=holders image

Will fix in another PR.

**Comment by FrederLu** [comment]
The size of the picture on the mobile terminal is not adapted.
Basemap：https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18
Picture ：https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10
image
image

**Comment by Keith-CY** [comment]
> The size of the picture on the mobile terminal is not adapted. Basemap：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18) Picture ：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10) image image

It could be tracked by another issue because the size of an NFT image is not related to .bit

**Comment by FrederLu** [comment]
> 
OK.
This pr modified the .bit avatar problem. If it is confirmed that it has nothing to do with this pr, open another issue to track.

**Comment by FrederLu** [comment]
> > The size of the picture on the mobile terminal is not adapted. Basemap：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0x0772997bfde614faa246b3cc98358f459d7911f7/18) Picture ：[godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10](https://godwoken-explorer-8afi0eh63-magickbase.vercel.app/nft-item/0xfec8e80db5d57b23fad94b828c88d716d3c6226b/10) image image
> 
> It could be tracked by another issue because the size of an NFT image is not related to .bit

OK.
This pr modified the .bit avatar problem. If it is confirmed that it has nothing to do with this pr, open another issue to track.

File: components/MultiTokenInventoryList/index.tsx [hunk 1]
```diff
@@ -14,7 +15,7 @@ type InventoryListProps = {
       token_id: string
       contract_address_hash: string
       counts: string
-      owner?: string
+      owner?: Record<string, any>
```
Comment (Keith-CY) [inline components/MultiTokenInventoryList/index.tsx:18]: Is the type of value unpredicatable?

File: components/MultiTokenInventoryList/index.tsx [hunk 2]
```diff
@@ -120,51 +121,58 @@ const MultiTokenInventoryList: React.FC<InventoryListProps> = ({ inventory, view
...
+                  <span>{(+item.counts || 0).toLocaleString('en')}</span>
+                </div>
+              )}
+
+              {token_id ? (
                 <div className={styles.info}>
                   <span>{t('owner')}</span>
-                  <HashLink href={`/account/${item.owner}`} label={item.owner} />
+                  <Address address={eth_address} domain={bit_alias} />
+                  {/* <HashLink href={`/account/${item.owner}`} label={item.owner} /> */}
```
Comment (Keith-CY) [inline components/MultiTokenInventoryList/index.tsx:160]: Unnecessary comment could be removed

**Comment by FrederLu** [comment]
> https://godwoken-explorer-2c6ln4cjd-magickbase.vercel.app/token/84440?tab=holders image Just to make sure, there are still some changes to be made here.

image
Except that the holder data interface in pr has not been replaced, so the .bit alias is not displayed, and will be modified in other pr later.
Everything else has been fixed.

**Review by FrederLu — APPROVED** [review]

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**